### ISSUE TITLE
New status purpose proposal

### ISSUE THREAD

**Issue by PatStLouis** [w3c/vc-bitstring-status-list#183]
Greetings, while implementing `BitstringStatusList` in our use case (supply-chain), we uncovered a situation where an issued credential might have a new version available for pickup which doesn't warrant a revocation of the previous version, but it would still be worthwhile to have a status to signal that an updated version is available.

Therefore I would like to suggest adding a status purpose ~`supersession`~ `refresh`, which can have a value of `0`(meaning this is the latest version of the credential available) or `1` (MAY be refreshed).

The process for a holder and/or verifier to get the latest version is out of scope, however I would intend this to be used in parallel with a ~`SupersessionRefresh`~ defined `refreshService` type, which contains an endpoint to query and get an updated version of the credential.

Once the software have retrieved the latest version of the credential, they would archive the current version and replace it with the latest version in their software, or create a [Symbolic link](https://en.wikipedia.org/wiki/Symbolic_link) towards it.

edit: renamed `supersession` to `refresh` as proposed.

**Comment by msporny** [comment]
+1 for this feature. 

Can we name the status purpose "refresh" instead of "supersession"? The former might be easier to understand.

**Comment by PatStLouis** [comment]
refresh would make sense, especially if its paired with a refresh service (optional, could also be out of bound). I had `update` at first then changed to `supersession` to align with the revocation/suspension terminologies.

Whichever term has consensus around which is representative of the features.

**Comment by msporny** [comment]
PR #185 has been raised to address this issue. This issue will be closed once PR #185 has been merged.

**Comment by msporny** [comment]
PR #185 has been merged, closing.
